app/models/fagus/item.rb

```diff
@@ -0,0 +1,35 @@
+# Fagus
+module Fagus
+  # Represents an item.
+  class Item < ActiveRecord::Base
+    extend Enumerize
+
+    belongs_to :category
+    belongs_to :item_type
+
+    enumerize :status,
+              :in => [:active, :phased_out, :discontinued],
+              :default => :active
+
+    validates \
+      :category,
+      :description,
+      :item_type,
+      :reference,
+      :status,
```

Comment:
After looking at the documentation for enumerize, it looks like it takes care of that....So never mind. Apologies for the run around.